tastyAPI.py

```diff
@@ -7,6 +7,7 @@
 from decimal import Decimal as D
 
 from dotenv import load_dotenv
+from helperAPI import Brokerage, maskString, printAndDiscord, printHoldings, stockOrder
```

Comment:
This doesn't need to move to a different spot

Reply:
I agree